config/migrations/20200907142800-procedurestap-delete-unused-properties.sparql

```diff
@@ -0,0 +1,77 @@
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX besluitvorming:  <http://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX dct: <http://purl.org/dc/terms/>
+
+DELETE{
+  GRAPH ?g {
+  ?ps ext:procedurestapNummer ?subcaseIdentifier
+  }
+}
+WHERE{
+    GRAPH ?g {
+        ?ps a dossier:Procedurestap .
+        ?ps ext:procedurestapNummer ?subcaseIdentifier .
+    }
+};
+
+DELETE{
+    GRAPH ?g {
+    ?ps ext:procedurestapNummer ?concluded
+    }
+}
+WHERE{
+    GRAPH ?g {
+        ?ps a dossier:Procedurestap .
+        ?ps besluitvorming:besloten ?concluded .
```

Comment:
Deze lijkt wel aanwezig op productie. Heb je in kaart gebracht waar precies, en of er geen waardevolle informatie verloren zal gaan indien we deze zonder meer verwijderen?

Reply:
```
PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
PREFIX besluitvorming:  <http://data.vlaanderen.be/ns/besluitvorming#>
PREFIX dct: <http://purl.org/dc/terms/>

SELECT * WHERE{
    GRAPH ?g {
        ?ps besluitvorming:besloten ?concluded .
    }
}
```
Geeft 33 resultaten (allemaal false) dus voor mij mag dit weg